bundles/org.smarthomej.binding.amazonechocontrol/README.md

```diff
@@ -178,6 +178,7 @@ You will find the serial number in the Alexa app or on the webpage YOUR_OPENHAB/
 | mediaLength               | Number:Time | R           | echo, echoshow, echospot      | Media length                                                                                                                                                                                                                            |
 | notificationVolume        | Dimmer      | R           | echo, echoshow, echospot      | Notification volume                                                                                                                                                                                                                     |
 | ascendingAlarm            | Switch      | R/W         | echo, echoshow, echospot      | Ascending alarm up to the configured volume                                                                                                                                                                                             |
+| dnd                       | Switch      | R/W         | echo, echoshow, echospot      | Do Not Disturb mode enabled                                        |
```

Comment:
Please name the channel `doNotDisturb`, it's hard to fond out what the channel is for if you use `dnd`.